subprojects/docs/src/docs/userguide/buildCache.adoc

```diff
@@ -161,7 +161,8 @@ Caching is then enabled for api:org.gradle.language.c.tasks.CCompile[] and api:o
 
 [NOTE]
 ====
-Caching native tasks is still experimental and there are some missing task inputs to the compilation tasks which can cause incorrect cache hits:
+Caching native tasks is still experimental.
```

Comment:
Let's remove the "still" here (it hasn't been experimental before), and then I think we're good.